private/src/scripts/modules/counter.js

```diff
@@ -19,7 +19,18 @@ const toFormattedString = (value) => {
   }
 
   const { currentLocale = 'en-GB' } = window.amnestyCoreI18n;
-  const formatted = toRawNumber(value).toLocaleString(currentLocale.replace('_', '-'));
+
+  const { forceTS } = window.amnestyForceTS;
+
+  let formatted = '';
+
+  if (forceTS === 'on') {
+    formatted = toRawNumber(value).toLocaleString(currentLocale.replace('_', '-'), {
+      useGrouping: true,
+    });
+  } else {
+    formatted = toRawNumber(value).toLocaleString(currentLocale.replace('_', '-'));
+  }
```

Comment:
The verbosity can be reduced by configuring the locale options, rather than the whole string:

```suggestion
  let options = null;
  if (forceTS) {
    options = {
      useGrouping: true,
    };
  }

  const formatted = toRawNumber(value).toLocaleString(currentLocale.replace('_', '-'), options);
```

Reply:
resolved